content/en/dashboards/reporting.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Reporting
+kind: documentation
+disable_toc: false
+is_public: true
+beta: true
+is_beta: true
+further_reading:
+  - link: "https://www.datadoghq.com/blog/template-variable-associated-values/"
+    tag: "Blog"
+    text: "Use associated template variables to refine your dashboards"
+  - link: "https://learn.datadoghq.com/course/view.php?id=8"
+    tag: "Learning Center"
+    text: "Building Better Dashboards"
+---
+
+<div class="alert alert-warning">This feature is currently in private beta. To request access, reach out to support@datadoghq.com.</div>
+
+## Overview
+
+Reporting automatically sends a visual summary of a dashboard to selected recipients on a schedule. The generated reports display images of widgets from a dashboard in a linear HTML format.
+
+{{< img src="dashboards/reporting/report_example.png" alt="An example report titled Checkout KPI Report showing a date, a description, a dashboard link, and 3 images of dashboard widgets" style="width:100%;" >}}
+
+## Schedule a report
+
+Create a new report from any dashboard by opening the dashboard’s cog menu and selecting **Schedule a Report**.
+
+The following widget types are supported:
+
+- Change
+- Distribution
+- Geomap
+- Group
+- Heat Map
+- Notes and Links
+- Query Value
+- Scatter Plot
+- Service Summary
+- Table
+- Timeseries
+- Top List
+
+In the configuration modal that opens, set a schedule for the report to determine when and how often the report is sent. Set a time frame to determine the range of time displayed in the resulting report. The report time frame can be different from the time frame displayed on the dashboard.
+
+Add recipients to your report by entering their email addresses. The email associated with your Datadog account is automatically added as a recipient. You can remove yourself as a recipient by hovering over your email and clicking the red **X**. 
+
+{{< img src="dashboards/reporting/report_configuration_modal.png" alt="The configuration modal for an individual dashboard report, with sections to set a schedule, add recipients, and customize email. At the bottom of the modal is a radio button to link to the live dashboard and other buttons to edit template variables, send preview, cancel, and save" style="width:100%;" >}}
```

Comment:
The "Link to the live dashboard" checkbox currently doesn't do anything, and is about to be removed -- I think it might be worth either waiting for this UI change, or manually editing the photo to not have it. (I'm a little worried that this might confuse customers.)

Reply:
The reporting modals are also about to have a header text update, we'll be able to make more accurate screenshots in a day or so. _(I don't think this distinction really matters from a documentation perspective but I thought I'd bring it up.)_